addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -17,14 +17,21 @@ const SelectUserValueWidget = options.userValueWidgetsRegistry['we-select'];
 
 const UrlPickerUserValueWidget = InputUserValueWidget.extend({
     custom_events: _.extend({}, InputUserValueWidget.prototype.custom_events || {}, {
-        website_url_chosen: '_onWebsiteURLChosen',
+        'website_url_chosen': '_onWebsiteURLChosen',
+    }),
+    events: _.extend({}, InputUserValueWidget.prototype.events || {}, {
+        'click .o_we_redirect_to': '_onRedirectTo',
     }),
 
     /**
      * @override
      */
     start: async function () {
         await this._super(...arguments);
+        const linkButton = document.createElement('we-button');
+        linkButton.classList.add('o_we_redirect_to', 'fa', 'fa-fw', 'fa-external-link');
+        linkButton.title = "Save and redirect to URL";
```

Comment:
Translation?

Reply:
Kinda afraid of saving then redirecting without any warning to the user, seems to be the opposite of what we do everywhere else.

@sbuhl In this case, should not we open in a new tab maybe? If not, then opening a modal to warn the user?

Reply:
@qsm-odoo Agree with you, the title isn't enough. We can use the modal to warn him.

Reply:
A modal with 3 choices ?
"This will save all your changes, are you sure you want to proceed?"
- OK
- New Window
- Cancel

Reply:
Would be easier to just use `Dialog.confirm` like `customizeWebsite`. Here I would open in a new tab... but if @sbuhl prefers the other way, then only a modal with ok, cancel.

Reply:
@sbuhl Opening a new tab without modal looks best to me then.